app/models/Post.scala

```diff
@@ -1,11 +1,12 @@
 package models
 
-import org.asciidoctor.Asciidoctor
+import org.asciidoctor.{Asciidoctor, Options}
 import org.asciidoctor.Asciidoctor.Factory
-import org.asciidoctor.ast.DocumentHeader
+import org.asciidoctor.ast.{Document, DocumentHeader}
```

Comment:
You can now cleanup the deprecated import too
```suggestion
import org.asciidoctor.ast.Document
```